src/styles/blocks/stars.css

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  justify-content: space-around;
+  align-items: center;
+  height: 116px;
+  padding: 22px;
+  width: 140px;
+}
```

Comment:
The class `.stars` is styled correctly for layout purposes, but ensure that the BEM methodology is consistently applied across all elements. The current class names like `.stars__inactive` and `.stars__active` should be used in conjunction with pseudo-selectors for better alignment with the task requirements.